src/app/containers/Blocks/index.jsx

```diff
@@ -16,6 +36,51 @@ const Blocks = ({ blocks, componentsToRender }) =>
 
     const Block = componentsToRender[type] || BlockString;
 
+    if (type === 'headline') {
```

Comment:
This would need to be logic that handles rendering anything before and up to the headline.